src/TLuaInterpreter.cpp

```diff
@@ -3179,12 +3122,7 @@ int TLuaInterpreter::saveProfile(lua_State* L)
         return 2;
     } else {
         auto message = QString("Couldn't save %1 to %2 because: %3").arg(host.getName(), std::get<1>(result), std::get<2>(result));
```

Comment:
:information_source: Needs to be switched from a `QString` to a `QStringLiteral` to match our coding style/guide IMHO.

Reply:
Maybe but not in scope really